From: Kent Overstreet <kent.overstreet@linux.dev>
Subject: fixup! fix missing vmalloc.h includes
Date: Wed, 3 Apr 2024 23:47:43 -0400

Few places were depending on sizes.h (convenience human-readable-units
size macros) iplicitly; fix

Link: https://lkml.kernel.org/r/20240327002152.3339937-1-kent.overstreet@linux.dev
Link: https://lkml.kernel.org/r/20240404034744.1664840-1-kent.overstreet@linux.dev
Signed-off-by: Kent Overstreet <kent.overstreet@linux.dev>
Cc: Suren Baghdasaryan <surenb@google.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/asm-generic/io.h |    1 +
 include/linux/io.h       |    1 +
 2 files changed, 2 insertions(+)

--- a/include/asm-generic/io.h~fix-missing-vmalloch-includes-fix-4
+++ a/include/asm-generic/io.h
@@ -9,6 +9,7 @@
 
 #include <asm/page.h> /* I/O is all done through memory accesses */
 #include <linux/string.h> /* for memset() and memcpy() */
+#include <linux/sizes.h>
 #include <linux/types.h>
 #include <linux/instruction_pointer.h>
 
--- a/include/linux/io.h~fix-missing-vmalloch-includes-fix-4
+++ a/include/linux/io.h
@@ -6,6 +6,7 @@
 #ifndef _LINUX_IO_H
 #define _LINUX_IO_H
 
+#include <linux/sizes.h>
 #include <linux/types.h>
 #include <linux/init.h>
 #include <linux/bug.h>
_
